HomeWork_04/HW4.py

```diff
@@ -0,0 +1,32 @@
+from datetime import time
+#---------------------#
+# Home Work 1
+
+words = input("Enter a text: ")
+
+for letter in words:
+    if letter.istitle():
+        print("This is a title")
+    elif letter.islower():
+        print("This is a lower")
+    elif letter.isdigit():
+        if (int(letter)) % 2 == 0:
+            print("This is a digit")
+        else:
+            print("This is not a digit")
```

Comment:
Here you will have a digit too, but it will be even. Please, update output information